Out-of-tree LLVM broken

```
05:12:52 DEBUG: Making all in llvm
05:12:52 DEBUG: which: no ninja in (/usr/lib/mono/llvm/bin:/opt/rh/devtoolset-2/root/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin:/builddir/bin)
05:12:52 DEBUG: make[2]: Entering directory `/builddir/build/BUILD/mono-5.17.0.282/llvm'
05:12:52 DEBUG: ../llvm/build_llvm_config.sh /usr/lib/mono/llvm/bin/llvm-config x86codegen > /builddir/build/BUILD/mono-5.17.0.282/llvm/llvm_config.mk
05:12:52 DEBUG: /bin/sh: ../llvm/build_llvm_config.sh: No such file or directory
05:12:52 DEBUG: make[2]: *** [/builddir/build/BUILD/mono-5.17.0.282/llvm/llvm_config.mk] Error 127
```

--- a/llvm/Makefile.am
+++ b/llvm/Makefile.am
@@ -1,5 +1,5 @@
 
-EXTRA_DIST=SUBMODULES.json build.mk
+EXTRA_DIST=SUBMODULES.json build.mk build_llvm_config.sh
 
 if ENABLE_LLVM
 if INTERNAL_LLVM

